src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Calibration/EditorExtrinsicsCalibration.cs

```diff
@@ -213,24 +175,55 @@ private void Update()
             }
 
             if (Input.GetKeyDown(KeyCode.Return))
+            {
+                CalculateExtrinsics();
+            }
+        }
+
+        public void RequestHeadsetData()
+        {
+            if (holographicCameraObserver != null &&
+                holographicCameraObserver.IsConnected)
+            {
+                using (MemoryStream memoryStream = new MemoryStream())
+                {
+                    using (BinaryWriter writer = new BinaryWriter(memoryStream))
+                    {

```

Comment:
Nit: you can place both of these using blocks together without each needing their own brace block, which is pattern used extensively elsewhere in the project.

using (MemoryStream memoryStream = new MemoryStream())
using (BinaryWriter writer = new BinaryWriter(memoryStream))
{
   ...
}